### DISCUSSION TITLE
Eclipse Theia v1.49.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13655] [Release Announcements]
Hi everyone :wave:,

The Eclipse Theia v1.49.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by jfaltermeier** [comment]
I think we might have an issue with Terminal Support at the moment, which could delay the release a bit. 

@msujew Terminal related tests have been failing since https://github.com/eclipse-theia/theia/issues/13613 :
https://github.com/eclipse-theia/theia/actions/workflows/playwright.yml
I saw that this is mentioned on the PR, but I think this is not an issue with the tests but an actual issue, since on Ubuntu the terminal does not work for me as well. I am on node version v20.12.2
How should we proceed here?

**Reply by jfaltermeier** [reply]
-> https://github.com/eclipse-theia/theia/issues/13657

**Reply by msujew** [reply]
I'm taking a look at this right now 👍

**Reply by msujew** [reply]
@jfaltermeier See https://github.com/eclipse-theia/theia/pull/13658 for a fix.

**Comment by jfaltermeier** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by jfaltermeier** [comment]
The release has completed, thank you to everyone that participated and contributed!

**Comment by tsmaeder** [comment]
@jfaltermeier I'm sorry to say I screwed up: I forgot to merge https://github.com/eclipse-theia/theia/pull/13611, which is pretty important for the Theia IDE case. I'm afraid we'll need to do a patch release. 😞

**Reply by jfaltermeier** [reply]
1.49.1 is available: https://github.com/eclipse-theia/theia/releases/tag/v1.49.1

**Comment by JonasHelming** [comment]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/05/08/eclipse-theia-1-49-release-news-and-noteworthy/)
